src/main/kotlin/nl/tudelft/ewi/se/ciselab/testgenie/settings/SettingsEvoSuiteComponent.kt

```diff
@@ -12,17 +12,22 @@ import javax.swing.JTextField
 /**
  * This class displays and captures changes to the values of the Settings entries.
  */
-class TestGenieSettingsComponent {
+class SettingsEvoSuiteComponent {
     var panel: JPanel? = null
-    private var showCoverageCheckBox = JCheckBox("Do you want visualised coverage? ")
-    private var sandboxCheckBox = JCheckBox("Execute tests in a sandbox environment")
-    private var assertionsCheckBox = JCheckBox("Create assertions")
-    private var seedTextField = JTextField()
-    //DropDown menu
+
+    //EvoSuite "input" options (e.g. text, number)
     private var algorithmSelector = ComboBox(ContentDigestAlgorithm.values())
     private var configurationIdTextField = JTextField()
-    private var clientOnThreadCheckBox = JCheckBox("Client on thread")
-    private var junitCheckCheckBox = JCheckBox("Compile and run suite")
+    private var seedTextField = JTextField()
```

Comment:
Looks better. Now it is easier to read and find the parameters you need

Reply:
nice